css/style.css

```diff
@@ -1,19 +1,140 @@
+.nav_bar {
+    background-color:whitesmoke;
+    position: fixed;
+    width: 100%;
+    border-bottom: 5px solid #DDD;
+}
+.nav_bar .nav_bar_ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+  
+  .nav_bar_ul li {
+      float: right;
+      align-content: center;
+      padding-inline-end: 20px;
+    }
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+  
+  .nav_bar_ul li a {
+    display: inline-block;
+    color: black;
+    text-align: center;
+    padding: 20px 10px;
+    text-decoration: none;
+  }
+  
+  .nav_bar_ul li a:hover:not(.active) {
+    background-color: green;
+  }
+  
+  .active {
+    font-style: italic;
+    background-color: rgb(146, 123, 123);
+  }
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
-}
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+  .logo-image{
+    display: block; 
+    height: 20px;
+    width: 50px;
+    margin-inline-start: 300px;}
+
+
+    .introduction{
+        display: block;
+        background-image: url(img/first-background.jpg);
```

Comment:
You can fix this like this:

```
    background: url('../img/first-background.jpg') 
```

Relative links can be a little tricky. You can learn about them here:
https://css-tricks.com/quick-reminder-about-file-paths/